modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -795,183 +798,233 @@
+                        "decision": null,
+                        "createdAt": "2013-01-17T16:54:11Z"
                       }
                     },
                     {
-                      "id": "600082088_3854594",
+                      "id": "111_11024",
                       "type": "powerOfAttorneyRequest",
                       "attributes": {
-                        "powerOfAttorneyCode": "074",
+                        "powerOfAttorneyCode": "003",
                         "isAddressChangingAuthorized": true,
                         "isTreatmentDisclosureAuthorized": true,
                         "veteran": {
-                          "firstName": "KENT",
-                          "middleName": "DAVID",
-                          "lastName": "RICHARDSON"
+                          "firstName": "[Vet First Name]",
+                          "middleName": null,
+                          "lastName": "[Vet Last Name]"
                         },
                         "claimant": null,
                         "claimantAddress": {
-                          "city": "HOUSTON",
-                          "state": "NM",
-                          "zip": "87110",
+                          "city": "Richmond",
+                          "state": null,
+                          "zip": "00123",
                           "country": "USA",
                           "militaryPostOffice": null,
                           "militaryPostalCode": null
                         },
-                        "decision": {
-                          "status": "Accepted",
-                          "declinedReason": null,
-                          "createdAt": "2024-04-04T18:10:56Z",
-                          "createdBy": {
-                            "firstName": "BEATRICE",
-                            "lastName": "STROUD",
-                            "email": "Beatrice.Stroud44@va.gov"
-                          }
-                        },
-                        "createdAt": "2024-04-04T18:08:05Z"
+                        "decision": null,
+                        "createdAt": "2013-01-17T17:04:22Z"
                       }
                     },
                     {
-                      "id": "600033542_3854600",
+                      "id": "111_11025",
                       "type": "powerOfAttorneyRequest",
                       "attributes": {
-                        "powerOfAttorneyCode": "074",
+                        "powerOfAttorneyCode": "003",
                         "isAddressChangingAuthorized": true,
                         "isTreatmentDisclosureAuthorized": true,
                         "veteran": {
-                          "firstName": "ALLAN",
+                          "firstName": "[Vet First Name]",
                           "middleName": null,
-                          "lastName": "BUTLER"
+                          "lastName": "[Vet Last Name]"
```

Comment:
are these supposed to be array's now?

Reply:
No, it's still a string. That was just entered in that way in staging data for some reason.

Reply:
Interesting. seems a bit misleading. any way we could find a better string to display?

Reply:
Ah good point that shows up in the docs. I'll hack the cassette to have something more presentable.

Reply:
@FonzMP I accidentally put the commit to make cleaner OAS examples in my followup branch, but that might be convenient anyway there were overlapping changes there anyway: https://github.com/department-of-veterans-affairs/vets-api/pull/17343/commits/2c966eda6e4d3527b8b91ff1ffe48f1acb764945

So could we merge this in and then move on to reviewing the final PR?